backend/src/mirrors_qa_backend/db/exceptions.py

```diff
@@ -0,0 +1,9 @@
+class ModelDoesNotExistError(Exception):
```

Comment:
`Model` is not the appropriate wording here. You're using it for missing `Record`.